source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,116 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
+
+* The encryption algorithm (deterministic or random)
+* The data key
+* The BSON Type
+
+.. note::
+
+    A single data key can be used for all encrypted fields, or you can use individual data keys that are specific to each field.
+
+
+The MedcoMD engineers receive specific requirements for the fields of data. `Deterministic Encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/#deterministic-encryption>`_ and `Random encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/#random-encryption>`_ are the two different methods of encrypting sensitive fields. The following table illustrates the data model of the Medco Management System.
+
+.. table:: 
+    :widths: 50 50
+
+    +-----------------------------+---------------------+----------------------------------------------+
+    |Field type                   |Encryption Algorithm | BSON Type                                    |
+    +=============================+=====================+==============================================+
+    | Name                        | Non-Encrypted       | String                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | SSN                         | Deterministic       |  Int                                         |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Blood Type                  | Random              |  String                                      |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Medical Records             | Random              |  Array                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Policy Number    | Deterministic       |  Int (embedded inside `insurance` object)    |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Provider         | Non-Encrypted       |  String (embedded inside `insurance` object) |
+    +-----------------------------+---------------------+----------------------------------------------+
+
+To automatically encrypt and decrypt create a JSON Schema using the information in the data model and pass the schema as a JSON object to the secure client to specify the encrypted field properties.
+In the JSON, you can specify `encryptMetadata <https://docs.mongodb.com/manual/reference/security-client-side-automatic-json-schema/#encryptmetadata-schema-keyword>`_ that is passed down to the fields under the ``properties``
+nested object. 
+
+.. code::
```

Comment:
We prefer to use a custom `code-block` directive instead of the `code` directive. It should work more or less the same as `code` but adds a copy button and some styling. You'll also need to specify the language of the code block (`none` is a valid language, if appropriate)

```
.. code-block:: json
   
   {
     ...
   }
```

Reply:
Fixed